README.md

```diff
@@ -40,7 +41,14 @@ modalForms: Ember.inject.service('modal-forms'),
 isModalActive: Ember.computed.readOnly('modalForms.isModalActive'),
 ```
 
-Change the active state of the modal in your component
+### Modal component
+Inject the [remodal](http://sethbrasile.github.io/ember-remodal/#/styling) and modal-forms services in your component
+```js
+remodal: inject.service(),
+modalForms: inject.service('modal-forms'),
```

Comment:
Unfortunately I don't believe the overlay is capable of a blur - the blur must be applied to the root application element in the DOM